### PR TITLE
FEAT Define Appendable to read input and error streams of process

### PR DESCRIPTION
**Pull request by mekahell**
Thanks for your amazing library :) I use it into one of my application, and each time FFmpeg process crashes we are not able to get the cause of its interruption. Theses modifications allow defining an Appendable instance to get input and error streams of process.

### CODE REVIEW

**Comment by barnettj** [comment]
Would be awesome if we could get this merged in.

**Comment by raphaelstolt** [comment]
Hello, is there some development on this valid pull request? Is there a way to assist you to get this merged and create a new release?

**Comment by Euklios** [comment]
@bramp: What's your thought on this?
I'd be willing to resolve the conflicts

File: src/test/java/net/bramp/ffmpeg/FFmpegTest.java [hunk 1]
```diff
@@ -8,6 +9,7 @@
 import org.junit.runner.RunWith;
 import org.mockito.Mock;
 import org.mockito.junit.MockitoJUnitRunner;
+import org.slf4j.event.Level;
```
Comment (bramp) [inline src/test/java/net/bramp/ffmpeg/FFmpegTest.java:12]: Unneeded?

File: src/test/java/net/bramp/ffmpeg/lang/NewProcessAnswer.java [hunk 2]
```diff
@@ -7,12 +7,21 @@
 public class NewProcessAnswer implements Answer<Process> {
   final String resource;
 
+  String errResource = null;
```
Comment (bramp) [inline src/test/java/net/bramp/ffmpeg/lang/NewProcessAnswer.java:10]: Should be final, to keep the class immutable

File: src/test/java/net/bramp/ffmpeg/FFmpegTest.java
Comment (bramp) [inline src/test/java/net/bramp/ffmpeg/FFmpegTest.java:1]: Thanks for the test!

**Comment by bramp** [comment]
I think it looks good. I've added feedback on the pr. Thanks

**Pull request closed without merging**